spot-ml/src/main/scala/org/apache/spot/lda/SpotLDAWrapper.scala

```diff
@@ -85,41 +80,67 @@ object SpotLDAWrapper {
         wordDictionary,
         sqlContext)
 
+    val corpusSize = ldaCorpus.count()
+
     docWordCountCache.unpersist()
 
     //Instantiate optimizer based on input
-    val optimizer = new EMLDAOptimizer
+    val ldaOptimizer = ldaOptimizerOption match {
+      case "em" => new EMLDAOptimizer
+      case "online" => new OnlineLDAOptimizer().setOptimizeDocConcentration(true).setMiniBatchFraction({
```

Comment:
(0.05+ 1) / corpus size I'm sure it's from https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/mllib/LDAExample.scala if corpus size < 2 then 0.75 I can't recall right now but I'm pretty sure that came up in a conversation with @Brandon.